web/src/containers/ApdPreviousActivityTableMMIS.js

```diff
@@ -26,23 +26,21 @@ const ApdPreviousActivityTableMMIS = ({
     <Fragment>
       <h4 className="ds-h4">MMIS</h4>
       {[90, 75, 50].map((level) => (
-        <table key={level} className="table-cms centered-headers table-fixed">
+        <table key={level} className="budget-table">
           <thead>
             <tr>
-              <th id="prev_act_mmis_null2" />
-              <th className="pre-line" id={`prev_act_mmis${level}_total`}>
+              <th id="prev_act_mmis_null2" rowSpan="2" />
```

Comment:
No real reason to include two null `th` cells when `rowspan` will do the trick with one!